sdk/core/azure-core-tracing-opentelemetry/src/test/java/com/azure/core/tracing/opentelemetry/OpenTelemetryHttpPolicyTests.java

```diff
@@ -102,6 +103,13 @@ private static HttpPipeline createHttpPipeline() {
         return httpPipeline;
     }
 
+    private static SpanContext getNonRemoteSpanContext(String diagnosticId) {
+        Context updatedContext = AmqpPropagationFormatUtil.extractContext(diagnosticId, Context.NONE);
+        SpanContext spanContext = (SpanContext) updatedContext.getData(SPAN_CONTEXT_KEY).get();
```

Comment:
Is `SPAN_CONTEXT_KEY` strongly known to exist in the Context? Based on the line of code above it seems like no, so should this use `Optional.orElse(null)` to return `null` if the key doesn't exist instead of throwing a NullPointerException?

Reply:
hey @alzimmermsft! `extractContext()` above populates `SPAN_CONTEXT_KEY` in the Context. I could add `orElseThrow()` if you think that's better (it's a failure case if it doesn't get populated for some reason). Though it's test code so my initial inclination is not to add guards 😄. Let me know, happy to make updates.

Reply:
I missed that this was under testing 😃 

I agree, let's not add guards here.